customisation/tokenisation.md

```diff
@@ -1,7 +1,26 @@
 ---
 grand_parent: Building and research
 parent: Customisation
-layout: coming_soon
 title: Tokenisation
-description:
+description: Splitting a string into a sequence of tokens
 ---
+
+Tokenisation is the process of splitting a string into a sequence of tokens.
+Tokens include words as well as punctuation characters.
+
+Example:
+- input: `"Yes, but why?"`
+- output: `["Yes", ",", "but", "why", "?"]`
+
+Machine translation algorithms learn the association between words in one language and words in another language.
+Tokenisation is needed because machine translation cannot operate effectively on the space of all possible strings, but can operate effectively on the space of all possible tokens.
+
+## Languages
+The most basic tokenisation algorithm is splitting the string on whitespace.
+Splitting on spaces is a reasonable baseline for many languages.
+However, there are several languages that do not separate words with whitespace, such as Chinese, Japanese, Korean, and Thai.
```

Comment:
I would rename this section as `## Challenges` and also mention agglutinative languages.

Reply:
Yeah that'd make sense, and would make it easier for me to introduce BPE. I'll try rephrasing the choice of tokenization algorithm to affect the vocabulary size, and therefore the curse of dimensionality, and therefore MT quality. I hope if I do that right, it'll be a natural transition into agglutinative, non-space-segmented languages, and others